src/screens/ReferFriends/ReferralContacts.js

```diff
@@ -234,9 +232,17 @@ class ReferralContacts extends React.PureComponent<Props, State> {
     } = this.props;
 
     const showConfirmButton = !!(selectedContacts.length || addedContactsToInvite.length);
-    const missingType = getInfoTypes(isPhoneVerified, isEmailVerified);
-    const allowedContacts = !missingType ? phoneContacts : phoneContacts.filter((contact) => !contact[missingType]);
-    const filteredContacts = getFilteredContacts(allowedContacts, query);
+    const allowedContacts = filterAllowedContacts(phoneContacts, isPhoneVerified, isEmailVerified);
+    const isSearching = query && query.length > MIN_QUERY_LENGTH;
```

Comment:
Per search block settings below:
```suggestion
    const isSearching = query && query.length >= MIN_QUERY_LENGTH;
```

Reply:
You're right! :+1: